### ISSUE TITLE
Business requirements - Adjust application to WCAG requirements 

### ISSUE THREAD

**Issue by maggienow** [binawork/OnlineOnboarding#251]
**User story**
Our application should be available to use by people with disabilities, and older people with changing abilities due to ageing/aging. 

**Tasks**

- [ ] Check solutions for implementing this WCAG requirements 
- [ ] Modify application according to requirements
- [ ] Test it by people with disabilities 
- [ ] WCAG  options implement for mobile and web version 

**DOD**
Solution implemented and tested by dedicated users
